api/python/provisioner/commands/check.py

```diff
@@ -623,7 +723,14 @@ def run(self, check_name: cfg.Checks = None,
         res: CheckResult = CheckResult()
 
         if check_name is None or check_name == cfg.Checks.ALL.value:
-            for check_name in cfg.CHECKS:
+            check_lists = cfg.SWUPDATE_CHECKS
```

Comment:
list of ALL != list of sw update, how we can request to do all checks then?

Reply:
Also, it was designed to allow the user running check by its name. For example,
```
provisioner check network
```
So we can allow running all checks by group name (sw_update, post_checks, replacenode_checks). But we need to track that names of granular checks will not intersect with group check names.

Reply:
now below command will run all checks available in CHECKS list

``` provisioner check all ```